### PR TITLE
change default work root dir to /var/lib/containerd-nydus

### PR DESCRIPTION
**Pull request by changweige**
Thus to keep consistent with /run/containerd-nydus

Signed-off-by: Changwei Ge <gechangwei@bytedance.com>

### CODE REVIEW

**Comment by liubogithub** [comment]
Hmm, if one had deployed an old version of nydus-snapshotter, after this upgrade, his setup won't work any more.

Can we leave some error messages to let users know that it's probably due to the change of root dir?

**Comment by changweige** [comment]
> Hmm, if one had deployed an old version of nydus-snapshotter, after this upgrade, his setup won't work anymore.
> 
> Can we leave some error messages to let users know that it's probably due to the change of root dir?

Reasonable. Fixed.

File: cmd/containerd-nydus-grpc/pkg/command/flags.go [hunk 1]
```diff
@@ -263,6 +264,14 @@ func Validate(args *Args, cfg *config.Config) error {
 		return errors.New("invalid empty root directory")
 	}
 
+	if args.RootDir == defaultRootDir {
+		if entries, err := os.ReadDir(oldDefaultRootDir); err == nil {
```
Comment (imeoer) [inline cmd/containerd-nydus-grpc/pkg/command/flags.go:268]: Old users will confuse if the error `default root directory is changed to /var/lib/containerd-nydus` be thrown and snapshotter exits, just give a warning for that?

Comment (changweige) [reply cmd/containerd-nydus-grpc/pkg/command/flags.go:269]: Changed to a warning message

**Review by imeoer — APPROVED** [review]

**Pull request merged**